src/index.js

```diff
@@ -0,0 +1,35 @@
+"use strict";
+
+import { nav_menu } from "./navbar";
+import { about_txt } from "./about";
+import { menu_txt } from "./menu";
+import { contact_txt } from "./contact";
+import { home_txt } from "./home";
+
+//-------inserts HOME PAGE text ---------------
+nav_menu();
+home_txt();
+//-----------------------------------------
+const home_link = document.querySelector("#home_link");
+home_link.addEventListener("click", e => {
+    home_txt();
+    e.preventDefault();
+});
+
+const about_link = document.querySelector("#about_link");
+about_link.addEventListener("click", e => {
```

Comment:
But isn't this repetition? Here is my suggestion:
- let the tab buttons all have the same click handler and pass something we can identify them by. let have our imported modules in an array, so depending on what is passed we render a different tab. that way we don't repeat.

Reply:
@SunnyAsar  Thanks for the suggestion, i had brought some amelioration to the code